### PR TITLE
Introducing the desktop user faq

### PR DESCRIPTION
**Pull request by chadyj**
For the alpha launch the desktop app gets a new `Need help?` menu item which links to this new desktop FAQ page.

### CODE REVIEW

File: source/docs/FAQ-desktop.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+Check [Github issues](https://github.com/status-im/status-react/labels/desktop) with the `desktop` label for any reported issues. If your issue is new please create a new issue and tag it `desktop`.
+
+It can also be reported in [#status](https://get.status.im/chat/public/status) in Status chat.
+
+## Where are Logs Located?
+
+Release builds do not log by default.  When enabled logs can be found here:
+
+### Mac 
+~/Library/Application Support/Status/Status.log
```
Comment (PombeirP) [inline source/docs/FAQ-desktop.md]:
```suggestion
`~/Library/Application Support/Status/Status.log`
```

File: source/docs/FAQ-desktop.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+~/Library/Application Support/Status/ethereum/mainnet_rpc/geth.log
```
Comment (PombeirP) [inline source/docs/FAQ-desktop.md]:
```suggestion
`~/Library/Application Support/Status/ethereum/mainnet_rpc/geth.log`
```

File: source/docs/FAQ-desktop.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+Download new release and make sure it’s named “Status.AppImage” and executable permissions are added to the file (chmod +x Status.AppImage).
+
+### Windows
+
+Download the new release, and run new installer as done on initial install.
+
+## What operating systems are supported?
+
+Linux, and MacOS, and Windows 7+.
```
Comment (PombeirP) [inline source/docs/FAQ-desktop.md]:
```suggestion
Linux, and MacOS, and Windows 10 (Windows 7+ support planned).
```

File: source/docs/FAQ-desktop.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+## How do I upgrade?
+
+### Mac
+
+Quit Status, download a new release, and drag to Applications
+
+### Linux 
+
+Download new release and make sure it’s named “Status.AppImage” and executable permissions are added to the file (chmod +x Status.AppImage).
```
Comment (PombeirP) [inline source/docs/FAQ-desktop.md]:
```suggestion
Download new release and make sure it’s named “Status.AppImage” and executable permissions are added to the file (`chmod +x Status.AppImage`).
```

File: source/docs/FAQ-desktop.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+App supports gnome-keyring or kwallet keychains (for db encryption key storage).
+
+To have gnome-keyring keychain working install following package:
+
+`sudo apt-get install libgnome-keyring0`
+
+Download AppImage; 
+if required, rename it to be exactly “Status.AppImage” (If you would have another name, app will start fresh without previously stored data).
+
+If you have high-DPI 4K resolution display, you should set QT_AUTO_SCREEN_SCALE_FACTOR, QT_SCREEN_SCALE_FACTORS, QT_SCALE_FACTOR env vars to work with your display’s dpi and resolution.
```
Comment (PombeirP) [inline source/docs/FAQ-desktop.md]:
```suggestion
If you have high-DPI 4K resolution display, you should set `QT_AUTO_SCREEN_SCALE_FACTOR`, `QT_SCREEN_SCALE_FACTORS`, `QT_SCALE_FACTOR` env vars to work with your display’s DPI and resolution.
```

File: source/docs/FAQ-desktop.md [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+
+### Linux
+
+prerequisites:
+
+App supports gnome-keyring or kwallet keychains (for db encryption key storage).
+
+To have gnome-keyring keychain working install following package:
+
+`sudo apt-get install libgnome-keyring0`
```
Comment (PombeirP) [inline source/docs/FAQ-desktop.md]:
```suggestion
``` bash
sudo apt-get install libgnome-keyring0
```
```

File: source/docs/FAQ-desktop.md [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+
+## I found a bug. Where do I report it?
+
+Check [Github issues](https://github.com/status-im/status-react/labels/desktop) with the `desktop` label for any reported issues. If your issue is new please create a new issue and tag it `desktop`.
+
+It can also be reported in [#status](https://get.status.im/chat/public/status) in Status chat.
+
+## Where are Logs Located?
+
+Release builds do not log by default.  When enabled logs can be found here:
```
Comment (PombeirP) [inline source/docs/FAQ-desktop.md]:
```suggestion
Release builds do not log by default. When enabled logs can be found here:
```

File: source/docs/FAQ-desktop.md [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+
+### Linux 
+~/.local/share/Status/Status.log
+~/.local/share/Status/ethereum/mainnet_rpc/geth.log
+
+### Windows
+%LOCALAPPDATA%\Status\Status.log
+%LOCALAPPDATA%\Status\ethereum\mainnet_rpc\geth.log
+
+
```
Comment (PombeirP) [inline source/docs/FAQ-desktop.md]:
```suggestion
```

**Comment by EugeOrtiz** [comment]
[Designs for reference here](https://www.figma.com/file/MRYBI5axcTwL4Ft8xAYhFOfy/Desktop?node-id=762%3A742) (Figma):

new profile - need help

**Comment by chadyj** [comment]
Thanks Pedro!  Changes applied.

**Comment by chadyj** [comment]
ooof didn't mean to merge/close this.  sorry

**Pull request merged**